### PR TITLE
Add support for GETDEL command

### LINKED ISSUE #1729: GETDEL support
I was wondering if there are plans to support the new GETDEL command? The docs mention that it was added in v6.2:

[](https://redis.io/commands/getdel)

Thanks!

### PR DESCRIPTION
**Pull request by WeihanLi**
Fixes #1729

### CODE REVIEW

File: src/StackExchange.Redis/Enums/RedisCommand.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ internal enum RedisCommand
         GETBIT,
         GETRANGE,
         GETSET,
+        GETDEL,
```
Comment (mgravell) [inline src/StackExchange.Redis/Enums/RedisCommand.cs:54]: Please move to be alphabetical

File: src/StackExchange.Redis/Message.cs [hunk 2]
```diff
@@ -360,6 +360,7 @@ public static bool IsMasterOnly(RedisCommand command)
                 case RedisCommand.FLUSHALL:
                 case RedisCommand.FLUSHDB:
                 case RedisCommand.GETSET:
+                case RedisCommand.GETDEL:
```
Comment (mgravell) [inline src/StackExchange.Redis/Message.cs:363]: Please move to be alphabetical

**Review by mgravell — DISMISSED** [review]
Looks fine, ta. Please could you add to the release notes md file, in the unreleased section, thanks. An integration test would be ideal, but I don't know if our CI redis servers offer this feature - would have to check.

**Comment by WeihanLi** [comment]
Thanks @mgravell, the command order had been updated to be alphabetical and the release notes had been added.
And is there some guide for adding the integration test?

**Comment by WeihanLi** [comment]
Hi @mgravell , the integration test had been added, any more changes needed?

File: src/StackExchange.Redis/Message.cs (hunk 2 above)
Comment (WeihanLi) [reply src/StackExchange.Redis/Message.cs:362]: Done

File: src/StackExchange.Redis/Enums/RedisCommand.cs (hunk 1 above)
Comment (WeihanLi) [reply src/StackExchange.Redis/Enums/RedisCommand.cs]: Updated

**Review by NickCraver — APPROVED** [review]
Thanks for the additions and the tweaks here - looking great!

**Pull request merged**